homeassistant/util/ssl.py

```diff
@@ -139,6 +147,14 @@ def client_context(
     return _SSL_CONTEXTS.get(ssl_cipher_list, _DEFAULT_SSL_CONTEXT)
 
 
+def create_client_context(
```

Comment:
Please do this change in a separate PR. We shouldn't change core utils in the same per as an integration change

Reply:
Ok, I opened #142653 and will rebase this once it is merged.